minimesos/src/main/java/com/containersol/minimesos/mesos/ClusterArchitecture.java

```diff
@@ -166,13 +164,18 @@ public ClusterContainers getContainers() {
 
         /**
          * Includes the default {@link ZooKeeperContainer} instance in the cluster
+         *
+         * @return TODO(mwl)
          */
         public Builder withZooKeeper() {
             return withZooKeeper(new ZooKeeperConfig());
         }
 
         /**
          * Be explicit about the version of the image to use.
+         *
+         * @param zooKeeperConfig TODO(mwl)
+         * @return TODO(mwl)
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS1135)